Integrations/FeedOffice365/FeedOffice365.yml

```diff
@@ -112,30 +105,31 @@ script:
       secret: false
     - auto: PREDEFINED
       default: false
-      defaultValue: 'IPS'
-      description: The indicator type. Can be "IPS", "URLS", or "Both". The default value is "IPS".
+      defaultValue: IPS
+      description: The indicator type. Can be "IPS", "URLS", or "Both". The default
+        value is "IPS".
       isArray: false
       name: indicator_type
       predefined:
-      - 'IPS'
-      - 'URLS'
-      - 'Both'
+      - IPS
+      - URLS
+      - Both
       required: false
       secret: false
     deprecated: false
     description: Gets indicators from the feed.
     execution: false
     name: get-indicators
```

Comment:
Need to update this command with the vendor prefix - make sure to update your main with the updated command name.